src/libraries/System.Net.WebSockets.Client/tests/WebSocketHelper.cs

```diff
@@ -83,6 +91,18 @@ public static class WebSocketHelper
                     cws.Options.KeepAliveInterval = keepAliveInterval;
                 }
 
+                if (invoker == null)
+                {
+                    cws.Options.ClientCertificates.Add(Test.Common.Configuration.Certificates.GetClientCertificate());
+                    cws.Options.RemoteCertificateValidationCallback = delegate { return true; };
+                }
+
+                if (!PlatformDetection.IsBrowser)
+                {
+                    cws.Options.HttpVersion = version;
+                    cws.Options.HttpVersionPolicy = HttpVersionPolicy.RequestVersionExact;
```

Comment:
Are our automatic version upgrade/downgrade tests not using this helper?

Reply:
They are not